bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/editor/SummernoteEditor.java

```diff
@@ -1,6 +1,6 @@
 package de.agilecoders.wicket.extensions.markup.html.bootstrap.editor;
 
-import de.agilecoders.wicket.extensions.markup.html.bootstrap.icon.FontAwesome5CssReference;
+import de.agilecoders.wicket.extensions.markup.html.bootstrap.icon.FontAwesomeCssReference;
```

Comment:
even that we drop FA4 I think it would be better to keep `5` in the name
I think it would be easier for the users to upgrade from Wicket-Bootstrap-4.x to upgrade to W-B-5.x

Reply:
will rename back in next PR :)